fastestimator/util/latex_util.py

```diff
@@ -129,3 +129,53 @@ def __init__(self,
             data.append(bold(escape_latex(name)) if bold_name else escape_latex(name))
         data.append(NoEscape("}}}"))
         super().__init__(data=data)
+
+
+class IterJoin(Container):
+    """A class to convert an iterable to a latex representation.
+
+    The data of this class can be any type. Usually it is iterable due to its capability to setup interval string.
+
+    Args:
+        data: Data of the cell.
+        token: String to be added in the interval of data entries.
+    """
+    def __init__(self, data: Any, token: str):
+        super().__init__(data=data)
+        self.token = token
+
+    def dumps(self) -> str:
+        """Get a string representation of this cell.
+
+        Returns:
+            A string representation of itself.
+        """
+        return dumps_list(self, token=self.token)
+
+
+class WrapText(LatexObject):
+    """A class to convert strings or numbers to wrappable latex representation.
+
+    This class will first convert the data to string, and then to wrappable latex representation if its length is too
+    long. This is to fix the issue that first string or number is not wrappable in X column type.
+
+    Args:
+        data: Input data to be converted.
+        threshold: When the length of <data> is greater than <threshold>, the resulting string will be made wrappable
```

Comment:
sorry, I think GitHub may have scrambled the feedback on doctstring formatting. Data and threshold should be quoted with the ` character rather than with <>